dev/lib/product_taxonomy/commands/generate_dist_command.rb

```diff
@@ -39,6 +41,16 @@ def execute
 
     private
 
+    def clear_shopify_integrations_directory
```

Comment:
I think this behaviour fits better in the `IntegrationVersion` class, since that's the one responsible for generating the integration files. `IntegrationVersion#generate_distributions` seems like a good place for it.

Another advantage of having it there is that we could reuse the logic for generating the integrations path and avoid multiple sources of truth for it.